broadcasting.md

```diff
@@ -748,9 +748,9 @@ If you would like to listen for events on a private channel, use the `private` m
 
 ```js
 Echo.private(`orders.${this.order.id}`)
-    .listen(...)
-    .listen(...)
-    .listen(...);
+    .listen(/* ... */)
+    .listen(/* ... */)
+    .listen(/* ... */);
```

Comment:
These aren't first class callables, being JS, but I adjusted these to match the rest of the docs.